modules/vf-graphql-holochain/queries/plan.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Top-level queries relating to Plan
+ *
+ * @package: HoloREA
+ * @since:   2019-05-27
+ */
+
+import { DNAIdMappings, injectTypename } from '../types'
+import { mapZomeFn } from '../connection'
+
+import {
+    Plan,
+    PlanConnection,
+} from '@valueflows/vf-graphql'
+
+export default (dnaConfig: DNAIdMappings, conductorUri: string) => {
+  const readOne = mapZomeFn(dnaConfig, conductorUri, 'planning', 'plan', 'get_plan')
+  const readAll = mapZomeFn(dnaConfig, conductorUri, 'planning', 'plan', 'get_all_plans')
```

Comment:
This will need to change to the `plan_index` zome, `read_all_plans` method, per incoming updates to #221.